### ISSUE TITLE
`constant literals have a type` error

### ISSUE THREAD

**Issue by SUPERCILEX** [cargo-public-api/cargo-public-api#583]
Run `UPDATE_EXPECT=1 TRYCMD=overwrite cargo t --workspace` inside https://github.com/SUPERCILEX/clipboard-history/tree/master/ringboard.

```
rustc 1.81.0-nightly (f1586001a 2024-06-13)
binary: rustc
commit-hash: f1586001ace26df7cafeb6534eaf76fb2c5513e5
commit-date: 2024-06-13
host: x86_64-unknown-linux-gnu
release: 1.81.0-nightly
LLVM version: 18.1.7
```

```
---- api stdout ----
thread 'api' panicked at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:797:50:
constant literals have a type
stack backtrace:
   0:     0x5e3354193cf5 - $$GT$6borrow17h8315fb660c6c68cfE
                               at s_for_arg::{closure_env#1}>/library/std/src/../../backtrace/src/backtrace/libunwind.rs:116:5
   1:     0x5e3354193cf5 - r<alloc::string::String>>>
                               at s_for_arg::{closure_env#1}>/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x5e3354193cf5 - ap_builder..builder..value_parser..ValueParser$GT$$GT$15into_resettable17h9cead4384fc04137E
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:68:5
   3:     0x5e3354193cf5 - 7b7335E
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x5e33541c108b - $LT$T$GT$$u20$as$u20$alloc..vec..spec_from_iter..SpecFromIter$LT$T$C$I$GT$$GT$9from_iter17hb6a4364456237dd4E
                               at s_for_arg::{closure_env#1}>/library/core/src/fmt/rt.rs:165:63
   5:     0x5e33541c108b - $C$_$GT$$u20$as$u20$core..ops..drop..Drop$GT$4drop17ha420d0f9bf84fbc2E
                               at s_for_arg::{closure_env#1}>/library/core/src/fmt/mod.rs:1168:21
   6:     0x5e33541906ff - p_in_place$LT$alloc..raw_vec..RawVec$LT$clap_builder..error..context..ContextValue$GT$$GT$17hd81145babc14bee4E
                               at s_for_arg::{closure_env#1}>/library/std/src/io/mod.rs:1835:15
   7:     0x5e3354193ace - lap_builder::util::flat_map::Iter<clap_builder::util::id::Id, clap_builder::parser::matches::matched_arg::MatchedArg>, clap_builder::parser::validator::{impl#0}::missing_required_error::{closure_env#1}>, clap_builder::parser::validator::{impl#0}::missing_required_error::{closure_env#2}, (), core::iter::traits::iterator::Iterator::find::check::{closure_env#0}<&clap_builder::util::id::Id, &mut clap_builder::parser::validator::{impl#0}::missing_required_error::{closure_env#3}>, core::ops::control_flow::ControlFlow<&clap_builder::util::id::Id, ()>>
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:47:5
   8:     0x5e3354193ace - clone<clap_builder::util::any_value::AnyValueId, clap_builder::builder::ext::BoxedExtension>
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:34:9
   9:     0x5e33541955b9 - Iterator<Item=clap_builder::builder::possible_value::PossibleValue>, alloc::alloc::Global>>
  10:     0x5e3354195228 - help_template12HelpTemplate16write_after_help17h28f549026b96cc10E
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:289:9
  11:     0x5e3354162caa - l#13}
                               at s_for_arg::{closure_env#1}>/library/alloc/src/boxed.rs:2076:9
  12:     0x5e3354162caa - ntrol_flow::ControlFlow<(), ()>, clap_builder::builder::command::{impl#4}::get_all_long_flag_aliases::{closure_env#0}, core::iter::traits::iterator::Iterator::any::check::{closure_env#0}<&str, clap_builder::builder::command::{impl#7}::long_flag_aliases_to::{closure_env#1}>>, core::ops::control_flow::ControlFlow<(), ()>>
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:137:21
  13:     0x5e3354195dff - atch17hb85ab67bf0930366E
                               at s_for_arg::{closure_env#1}>/library/alloc/src/boxed.rs:2076:9
  14:     0x5e3354195dff - empty17he6d86253bb8f69f1E
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:801:13
  15:     0x5e3354195a27 - ap_builder::parser::features::suggestions::did_you_mean::{closure_env#2}<&str, core::iter::adapters::map::Map<core::slice::iter::Iter<alloc::string::String>, clap_builder::parser::parser::{impl#2}::did_you_mean_error::{closure_env#1}>>>, alloc::slice::stable_sort::{closure_env#0}<(f64, alloc::string::String), alloc::slice::{impl#0}::sort_by::{closure_env#0}<(f64, alloc::string::String), clap_builder::parser::features::suggestions::did_you_mean::{closure_env#2}<&str, core::iter::adapters::map::Map<core::slice::iter::Iter<alloc::string::String>, clap_builder::parser::parser::{impl#2}::did_you_mean_error::{closure_env#1}>>>>, alloc::slice::stable_sort::{closure_env#1}<(f64, alloc::string::String), alloc::slice::{impl#0}::sort_by::{closure_env#0}<(f64, alloc::string::String), clap_builder::parser::features::suggestions::did_you_mean::{closure_env#2}<&str, core::iter::adapters::map::Map<core::slice::iter::Iter<alloc::string::String>, clap_builder::parser::parser::{impl#2}::did_you_mean_error::{closure_env#1}>>>>, alloc::slice::stable_sort::{closure_env#2}<(f64, alloc::string::String), alloc::slice::{impl#0}::sort_by::{closure_env#0}<(f64, alloc::string::String), clap_builder::parser::features::suggestions::did_you_mean::{closure_env#2}<&str, core::iter::adapters::map::Map<core::slice::iter::Iter<alloc::string::String>, clap_builder::parser::parser::{impl#2}::did_you_mean_error::{closure_env#1}>>>>, alloc::slice::stable_sort::{closure_env#3}<(f64, alloc::string::String), alloc::slice::{impl#0}::sort_by::{closure_env#0}<(f64, alloc::string::String), clap_builder::parser::features::suggestions::did_you_mean::{closure_env#2}<&str, core::iter::adapters::map::Map<core::slice::iter::Iter<alloc::string::String>, clap_builder::parser::parser::{impl#2}::did_you_mean_error::{closure_env#1}>>>>>
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:667:13
  16:     0x5e33541941b9 - const$u20$T$GT$7sub_ptr17h61ab47adb980ddd1E
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:171:18
  17:     0x5e33541956d4 - }::all_subcommand_names::{closure_env#0}>>>>>
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:658:5
  18:     0x5e33541be263 - T$9into_iter17h485226ade7c8b2e8E
                               at s_for_arg::{closure_env#1}>/library/core/src/panicking.rs:74:14
  19:     0x5e33541be1ab - rser::{impl#32}::parse_ref::{closure_env#1}<i64>>
                               at s_for_arg::{closure_env#1}>/library/core/src/panicking.rs:264:5
  20:     0x5e33541be1ab - nv#2}<&str, core::iter::adapters::flatten::FlatMap<core::slice::iter::Iter<clap_builder::builder::command::Command>, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}>>>>>
                               at s_for_arg::{closure_env#1}>/library/core/src/option.rs:2021:5
  21:     0x5e3353c938f7 - core::option::Option<T>::expect::h8abfbbb16f10f8fc
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/option.rs:924:21
  22:     0x5e3353c7a4ba - public_api::render::RenderingContext::render_constant::hdf315f36ad54913b
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:797:44
  23:     0x5e3353c79f8e - public_api::render::RenderingContext::render_generic_arg::hf9c3664b9d44bb83
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:774:37
  24:     0x5e3353c79c3b - public_api::render::RenderingContext::render_angle_bracketed::{{closure}}::h990c832b18a1250b
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:751:41
  25:     0x5e3353c71ae8 - public_api::render::RenderingContext::render_sequence_impl::h23ddfc2fc2456414
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:321:27
  26:     0x5e3353c7183f - public_api::render::RenderingContext::render_sequence_if_not_empty::he43d8f4612e544d9
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:304:9
  27:     0x5e3353c79ac6 - public_api::render::RenderingContext::render_angle_bracketed::h6b7da867a9ae612a
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:741:9
  28:     0x5e3353c792fa - public_api::render::RenderingContext::render_generic_args::h56cee51a0d304e11
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:713:17
  29:     0x5e3353c76813 - public_api::render::RenderingContext::render_resolved_path::h5809a7fd82a65158
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:535:27
  30:     0x5e3353c7404b - public_api::render::RenderingContext::render_option_type::hd312a682dd5d5c42
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:343:41
  31:     0x5e3353c7367b - public_api::render::RenderingContext::render_type::hb94e0dc412f6e3f5
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:331:9
  32:     0x5e3353c70124 - public_api::render::RenderingContext::token_stream::h5145d241ca791b45
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/render.rs:110:31
  33:     0x5e3353cb3028 - public_api::intermediate_public_item::IntermediatePublicItem::render_token_stream::he6c1cdce5e72d23d
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/intermediate_public_item.rs:52:9
  34:     0x5e3353c8f184 - public_api::public_item::PublicItem::from_intermediate_public_item::h086fd42ae9a1c18e
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/public_item.rs:36:21
  35:     0x5e3353c8f12c - public_api::item_processor::public_api_in_crate::{{closure}}::h7637be142dbf8aa5
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/item_processor.rs:463:25
  36:     0x5e3353c92ade - core::iter::adapters::map::map_fold::{{closure}}::h7a92fc743e91a2a0
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/iter/adapters/map.rs:89:28
  37:     0x5e3353c6d129 - <core::slice::iter::Iter<T> as core::iter::traits::iterator::Iterator>::fold::h7b0123320757eeea
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/slice/iter/macros.rs:230:27
  38:     0x5e3353c8fb9e - <core::iter::adapters::map::Map<I,F> as core::iter::traits::iterator::Iterator>::fold::hfb046d4cc9efc40a
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/iter/adapters/map.rs:129:9
  39:     0x5e3353c92562 - core::iter::traits::iterator::Iterator::for_each::hfb80f08d25d44a4a
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/iter/traits/iterator.rs:818:9
  40:     0x5e3353c5232a - alloc::vec::Vec<T,A>::extend_trusted::h3b7636391fb24bff
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/alloc/src/vec/mod.rs:3096:17
  41:     0x5e3353c583aa - <alloc::vec::Vec<T,A> as alloc::vec::spec_extend::SpecExtend<T,I>>::spec_extend::h8bdd868c6301c22a
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/alloc/src/vec/spec_extend.rs:26:9
  42:     0x5e3353c4f3e9 - <alloc::vec::Vec<T> as alloc::vec::spec_from_iter_nested::SpecFromIterNested<T,I>>::from_iter::h1ee4a4081c551fb5
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/alloc/src/vec/spec_from_iter_nested.rs:62:9
  43:     0x5e3353c584a4 - <alloc::vec::Vec<T> as alloc::vec::spec_from_iter::SpecFromIter<T,I>>::from_iter::h4bd267d9ff5c0526
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/alloc/src/vec/spec_from_iter.rs:33:9
  44:     0x5e3353c57fe4 - <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter::h235228a34209de6b
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/alloc/src/vec/mod.rs:2970:9
  45:     0x5e3353c920e8 - core::iter::traits::iterator::Iterator::collect::h73ef073ff5f38dd6
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/iter/traits/iterator.rs:2005:9
  46:     0x5e3353c8f081 - public_api::item_processor::public_api_in_crate::h25f66862195fa4af
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/item_processor.rs:460:16
  47:     0x5e3353c9a957 - public_api::from_rustdoc_json_str::h2c5242947602b1bd
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/lib.rs:265:26
  48:     0x5e3353c9a420 - public_api::Builder::build::h98e38158b3b2b76b
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/public-api-0.35.0/src/lib.rs:177:9
  49:     0x5e33538f24b5 - supercilex_tests::api::hc7c4f2cfb0941510
                               at /home/asaveau/.cargo/registry/src/index.crates.io-6f17d22bba15001f/supercilex-tests-0.4.5/src/lib.rs:82:19
  50:     0x5e33538eec78 - api::api::hf73c5f487a60a4ff
                               at /home/asaveau/Desktop/ringboard/ringboard/tests/api.rs:3:5
  51:     0x5e33538eede9 - api::api::{{closure}}::h93398a7c4ae6458d
                               at /home/asaveau/Desktop/ringboard/ringboard/tests/api.rs:2:9
  52:     0x5e33538eefb1 - core::ops::function::FnOnce::call_once::hb348a47698a951b9
                               at /rustc/f1586001ace26df7cafeb6534eaf76fb2c5513e5/library/core/src/ops/function.rs:250:5
  53:     0x5e33541674fb - s$u20$clap_builder..builder..value_parser..TypedValueParser$GT$9parse_ref28_$u7b$$u7b$closure$u7d$$u7d$17h6075e68b1863181fE
                               at s_for_arg::{closure_env#1}>/library/core/src/ops/function.rs:250:5
  54:     0x5e33541674fb - P$$RP$$RP$$GT$17h55ebac88cee1438cE
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:625:18
  55:     0x5e3354166c51 - or$LT$T$GT$$GT$9from_iter17h6017b5798e665bbdE
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:648:60
  56:     0x5e3354166c51 - ched_arg::MatchedArg>, clap_builder::parser::validator::{impl#0}::missing_required_error::{closure_env#1}>, &clap_builder::util::id::Id, clap_builder::parser::validator::{impl#0}::missing_required_error::{closure_env#2}>
                               at s_for_arg::{closure_env#1}>/library/core/src/panic/unwind_safe.rs:272:9
  57:     0x5e3354166c51 - ut17h942a5faf2a0f4597E
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:553:40
  58:     0x5e3354166c51 - c3vec16Vec$LT$T$C$A$GT$8is_empty17h69d3b62c2bfc04ceE
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:517:19
  59:     0x5e3354166c51 - ::MountPropagationFlags>>
                               at s_for_arg::{closure_env#1}>/library/std/src/panic.rs:350:14
  60:     0x5e3354166c51 - c$LT$T$C$A$GT$$u20$as$u20$core..ops..deref..Deref$GT$5deref17h08b9e6962188e9e9E
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:648:27
  61:     0x5e3354166c51 - at_map..Iter$LT$K$C$V$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$4next17hfc2ba93b20267fe9E
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:569:43
  62:     0x5e335412f2d4 - ry_fold::{closure_env#0}<&clap_builder::builder::command::Command, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}, core::iter::adapters::flatten::{impl#33}::iter_try_fold::flatten::{closure_env#0}<core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, core::iter::adapters::flatten::{impl#35}::try_fold::flatten::{closure_env#0}<core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, core::iter::adapters::map::map_try_fold::{closure_env#0}<&str, (f64, alloc::string::String), (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, clap_builder::parser::features::suggestions::did_you_mean::{closure_env#0}<&str, core::iter::adapters::flatten::FlatMap<core::slice::iter::Iter<clap_builder::builder::command::Command>, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}>>, core::iter::traits::iterator::Iterator::find::check::{closure_env#0}<(f64, alloc::string::String), &mut clap_builder::parser::features::suggestions::did_you_mean::{closure_env#1}<&str, core::iter::adapters::flatten::FlatMap<core::slice::iter::Iter<clap_builder::builder::command::Command>, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}>>>>>>>, core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>>
                               at s_for_arg::{closure_env#1}>/library/test/src/lib.rs:599:41
  63:     0x5e335412f2d4 - nce::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, core::iter::adapters::flatten::{impl#35}::try_fold::flatten::{closure_env#0}<core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, core::iter::adapters::map::map_try_fold::{closure_env#0}<&str, (f64, alloc::string::String), (), core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>, clap_builder::parser::features::suggestions::did_you_mean::{closure_env#0}<&str, core::iter::adapters::flatten::FlatMap<core::slice::iter::Iter<clap_builder::builder::command::Command>, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}>>, core::iter::traits::iterator::Iterator::find::check::{closure_env#0}<(f64, alloc::string::String), &mut clap_builder::parser::features::suggestions::did_you_mean::{closure_env#1}<&str, core::iter::adapters::flatten::FlatMap<core::slice::iter::Iter<clap_builder::builder::command::Command>, core::iter::adapters::chain::Chain<core::iter::sources::once::Once<&str>, core::iter::adapters::map::Map<core::slice::iter::Iter<(clap_builder::builder::str::Str, bool)>, clap_builder::builder::command::{impl#4}::get_all_aliases::{closure_env#0}>>, clap_builder::builder::command::{impl#7}::all_subcommand_names::{closure_env#0}>>>>>>, core::ops::control_flow::ControlFlow<(f64, alloc::string::String), ()>>
                               at s_for_arg::{closure_env#1}>/library/std/src/sys_common/backtrace.rs:155:18
  64:     0x5e3354133bc2 - y_value::AnyValue, alloc::alloc::Global>
                               at s_for_arg::{closure_env#1}>/library/std/src/thread/mod.rs:542:17
  65:     0x5e3354133bc2 - 845c925E
                               at s_for_arg::{closure_env#1}>/library/core/src/panic/unwind_safe.rs:272:9
  66:     0x5e3354133bc2 - r$LT$T$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$4next17h682b360f19e08e3dE
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:553:40
  67:     0x5e3354133bc2 - oIterator$GT$9into_iter17hd73741c7721d0529E
                               at s_for_arg::{closure_env#1}>/library/std/src/panicking.rs:517:19
  68:     0x5e3354133bc2 - $u20$$RF$mut$u20$F$GT$8call_mut17h94e673aabd498dc6E
                               at s_for_arg::{closure_env#1}>/library/std/src/panic.rs:350:14
  69:     0x5e3354133bc2 - sure_env#6}>>, core::ops::control_flow::ControlFlow<&clap_builder::builder::arg::Arg, ()>>
                               at s_for_arg::{closure_env#1}>/library/std/src/thread/mod.rs:541:30
  70:     0x5e3354133bc2 - r7command7Command11two_args_of17h583ed5b48cb8e65bE
                               at s_for_arg::{closure_env#1}>/library/core/src/ops/function.rs:250:5
  71:     0x5e335419b2fb - s..types.._..InternalBitFlags$u20$as$u20$core..fmt..Debug$GT$3fmt17h7b002056e0522d63E
                               at s_for_arg::{closure_env#1}>/library/alloc/src/boxed.rs:2062:9
  72:     0x5e335419b2fb -  core::fmt::Formatter>
                               at s_for_arg::{closure_env#1}>/library/alloc/src/boxed.rs:2062:9
  73:     0x5e335419b2fb - rator$GT$4next17h44d23fb18c06aa55E
                               at s_for_arg::{closure_env#1}>/library/std/src/sys/pal/unix/thread.rs:108:17
  74:     0x7c291f094ac3 - start_thread
                               at ./nptl/pthread_create.c:442:8
  75:     0x7c291f126850 - __GI___clone3
                               at ./misc/../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
  76:                0x0 - <unknown>


failures:
    api
```

**Comment by Enselic** [comment]
Thanks for reporting, but I can't reproduce:
```console
$ UPDATE_EXPECT=1 TRYCMD=overwrite cargo t --workspace
error: failed to load manifest for workspace member `/home/martin/src/clipboard-history/server`
referenced by workspace at `/home/martin/src/clipboard-history/Cargo.toml`

Caused by:
  failed to load manifest for dependency `io-uring`

Caused by:
  failed to read `/home/asaveau/Desktop/wip/io-uring/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```

Can you minimize the reproducer please? You are hitting

https://github.com/Enselic/cargo-public-api/blob/f740ac591b07475c47bb0908d0f763a1550570ee/public-api/src/render.rs#L797

and the fix is probably easy, but we need a regression test.

If it's hard to minimize, it would still be helpful with exact reproducability tests.  You might find some inspiration in our [minimal dev env](https://github.com/Enselic/cargo-public-api/blob/main/docs/MAINTAINER.md#minimal-dev-env) docs.

**Comment by SUPERCILEX** [comment]
Oh crap, sorry about the local dependencies. Honestly forgot I had that. Will fix.

**Comment by SUPERCILEX** [comment]
Fixed if you pull. Seems to be mad at this line: https://github.com/SUPERCILEX/clipboard-history/blob/054f2c127bf4c551c66d2ffa8026642bcbdd1ef9/ringboard/src/protocol.rs#L19

**Comment by Enselic** [comment]
Minimal reproducer integrated with the test suite: https://github.com/Enselic/cargo-public-api/pull/584

I'm quite busy the coming days, so not sure I get time to fix this right way. If someone wants to take a shot, feel free. Feel free to steal my commit in that PR without attribution. The fix might be easy.

Run

    ./scripts/bless-expected-output-for-tests.sh

after fixing.

**Comment by Enselic** [comment]
I got some time over and updated the PR with a [fix](https://github.com/Enselic/cargo-public-api/pull/584). I will likely merge and release in a couple of days or so. (It will be released as v0.36.0 according to our versioning strategy: https://github.com/Enselic/cargo-public-api/blob/main/docs/RELEASE.md#versioning-strategy)

**Comment by SUPERCILEX** [comment]
Sweet, thank you!

**Comment by Enselic** [comment]
I changed the fix to be 0.0.x compatible and just released it in 0.35.1

**Comment by SUPERCILEX** [comment]
Neat, thanks!
